### PR TITLE
DATA SCIENCE TEAM - Tree History - 100% Complete

### PR DESCRIPTION
**Pull request by hiruboy96**
_**Use case completed**_

Use case includes:
- Adding Description
- Adding Title
- Loaded Data through API
- Removing Unwanted columns
- Renaming columns to match the other dataset
- Historical vs. Current Species Diversity (Venn Diagram)
- Change in Species Abundance Over Time
- Geographical Distribution of Trees (Map Visualization)
- Time-Series Analysis of Tree Planting
- Habitat Analysis for Pre-Colonial Species
- Ecological Dynamics of Tree Species in Parks vs. Streets in Melbourne
- Adding filters to the map
- Timeseries Model Fitting (ARIMA Model Forecasting)
- Forecasted tree plantings for the future 5 years
- Updated the API to catch the data by hiding the API key using Environment Variable.
- Finalizing the use case by achieving the desired outcome.

### CODE REVIEW

**Comment by HarleyNgo** [comment]
Hi Venuka!
Would you mind checking again your file? As from my side, it stops at map visualisation and I'm sure that's not the end of the usecase.

**Review by HarleyNgo — CHANGES_REQUESTED** [review]
please upload the latest file

**Comment by hiruboy96** [comment]
[Tree History](https://trello.com/c/aWVrUG1r/442-tree-history)

**Review by AliColli — COMMENTED** [review]
Hi Venuka, your use case opens fine in https://nbviewer.org/
For your scenario, write this use case so that it is for other people. E.g. as a conservationist employed by the city of Melbourne. Or as  city planner, Or as a leader of the City of Melbourne Urban Forest Strategy Project .... 
Great explanation of why preprocessing steps were included.
Historical vs. Current Species Diversity Venn Diagram needs an explanation of what it shows. You have explained how the code works but not what the graph shows. Add a couple of sentences to explain what is known from this Venn Diagram about tree species.
Change in Species Abundance Over Time print out needs to be explained. How many are gained and lost? This needs to be in words as well.
Geographical Distribution of Trees (Map Visualization) again missing an explanation of what the data shows. You need to explain what the numbers are on the graph and what happens as you zoom in and out. What is the user seeing from the map?
Time-Series Analysis of Tree Planting again missing explanation of graph. Which years have the most and which the least? Explanation of trends? e.g. covid
Habitat Analysis for Pre-Colonial Species need to explain what the insights are, not just that there are insights :)
Ecological Dynamics of Tree Species in Parks vs. Streets in Melbourne explain which has more, which has less. Compare the two with words.
Identify Lost Species make sure you state how many species are lost as well.  What is the purpose of printing the shape of each dataset in this section? Please explain in markdown.
Timeseries Model Fitting (ARIMA Model Forecasting) Make sure that you explain what is output from each step. Explain what the graph shows, what the stationary means, the results of the model fit summary (was it a good fit?), forecasting results summary in a sentence, what MAE and RMSE results mean.
In conclusions, make sure that you answer the aim of your use case - how many trees should be planted in future years?
In summary, excellent explanations of processes used but need more emphasis on results and their explanation.

**Comment by hiruboy96** [comment]
Closing due to code issues.
